src/OSPSuite.Core/Domain/Services/ParameterIdentifications/ParameterIdentificationRunner.cs

```diff
@@ -23,9 +23,9 @@ public class ParameterIdentificationRunner : IParameterIdentificationRunner
       private readonly IDialogCreator _dialogCreator;
       private readonly IEntityValidationTask _entityValidationTask;
       private readonly IOSPSuiteExecutionContext _executionContext;
-      private IParameterIdentificationEngine _parameterIdentificationEngine;
+      private readonly Cache<ParameterIdentification, IParameterIdentificationEngine> _parameterIdentificationEngines = new Cache<ParameterIdentification, IParameterIdentificationEngine>(onMissingKey: x => null);
 
-      public bool IsRunning => _parameterIdentificationEngine != null;
+      public bool IsRunning => _parameterIdentificationEngines.Count > 0;
```

Comment:
you could use Any() instead of count. Does not matter here but it's more idiomatic...maybe

Reply:
Sure, it is more idiomatic ;)